components/scream/src/physics/rrtmgp/tests/rrtmgp_unit_tests.cpp

```diff
@@ -403,3 +403,18 @@ TEST_CASE("rrtmgp_test_compute_broadband_surface_flux") {
     sfc_flux_dif_vis.deallocate();
     if (yakl::isInitialized()) { yakl::finalize(); }
 }
+
+TEST_CASE("rrtmgp_test_radiation_do") {
+
+    //
+    REQUIRE(scream::rrtmgp::radiation_do(2, 0) == true);
+    REQUIRE(scream::rrtmgp::radiation_do(2, 1) == false);
+    REQUIRE(scream::rrtmgp::radiation_do(2, 2) == true);
+    REQUIRE(scream::rrtmgp::radiation_do(2, 3) == false);
+
+    REQUIRE(scream::rrtmgp::radiation_do(3, 0) == true);
+    REQUIRE(scream::rrtmgp::radiation_do(3, 1) == false);
+    REQUIRE(scream::rrtmgp::radiation_do(3, 2) == false);
+    REQUIRE(scream::rrtmgp::radiation_do(3, 3) == true);
+    REQUIRE(scream::rrtmgp::radiation_do(3, 4) == false);
```

Comment:
might be worth adding a `radiation_do(3,6) == true` check.  To test when nsteps > the rad frequency and we want to update radiation.